public/app/features/dashboard/dashgrid/PanelHeader/PanelHeaderMenuProvider.tsx

```diff
@@ -24,6 +25,7 @@ export function PanelHeaderMenuProvider({ panel, dashboard, loadingState, childr
 
   useEffect(() => {
     setItems(getPanelMenu(dashboard, panel, angularComponent));
+    reportInteraction('dashboards_panelheader_menu', { item: 'menu' });
```

Comment:
Is this going to report an interaction every time we change any of the `useEffect` dependencies?

Reply:
@ivanortegaalba I changed the approach, now reporting the menu interaction only happens when the user opens the menu and sees the items. Can you take a look? 🙃